config.json

```diff
@@ -1971,6 +1971,20 @@
           "numbers"
         ],
         "difficulty": 5
+      },
+      {
+        "slug": "mazy-mice",
+        "name": "Mazy Mice",
+        "uuid": "1bac7473-9ee8-4cfc-928b-77792102ffc1",
+        "practices": [
+          "chars",
+          "arrays"
+        ],
```

Comment:
The `practices` array can (and IMO should) also contain the concepts that are listed in `prerequisites`, as this exercise definitely helps students practice `chars` and `arrays`.

Reply:
fixed